### PR TITLE
Some corrections to the text

### PR DESCRIPTION
**Pull request by bytemare**
As I was rereading the draft, I saw a couple of mistakes and missing spaces. So I ran the text through a corrector that found some other suggestions.

### CODE REVIEW

File: draft-irtf-cfrg-opaque.md [hunk 1]
```diff
@@ -318,9 +318,9 @@ pre-computation attacks (as defined in {{JKX18}}). OPAQUE provides forward
 secrecy with respect to password leakage while also hiding the password from
 the server, even during password registration. OPAQUE allows applications
 to increase the difficulty of offline dictionary attacks via iterated
-hashing or other key stretching schemes. OPAQUE is also extensible, allowing
+hashing or other key-stretching schemes. OPAQUE is also extensible, allowing
 clients to safely store and retrieve arbitrary application data on servers
-using only their password.
+using only their passwords.
```
Comment (chris-wood) [inline draft-irtf-cfrg-opaque.md:323]:
```suggestion
using only their password.
```

File: draft-irtf-cfrg-opaque.md [hunk 2]
```diff
@@ -2003,8 +2003,8 @@ implementation considerations.
   attack. This implementation only affects the server and changes nothing for the client.
   Furthermore, if the threshold OPRF servers holding these keys are separate from
   the authentication server, then recovering all n shares would still not suffice
-  to run an offline dictionnary attack without access to the client record database.
-  However, this mechanism is out of scope for this document.
+  to run an offline dictionary attack without access to the client record database.
+  However, this mechanism is out of the scope of this document.
```
Comment (chris-wood) [inline draft-irtf-cfrg-opaque.md:2007]:
```suggestion
  However, this mechanism is out of scope for this document.
```

File: draft-irtf-cfrg-opaque.md [hunk 3]
```diff
@@ -2249,7 +2249,7 @@ between client and server during registration, e.g., using TLS {{RFC8446}}.
 If the channel is only authenticated (this is a requirement for correct
 identification of the parties), a confidential channel can be established
 using public-key encryption, e.g., with HPKE {{?RFC9180}}. However, the details
-of this mechanism are out of scope of this document.
+of this mechanism are out of the scope of this document.
```
Comment (chris-wood) [inline draft-irtf-cfrg-opaque.md:2252]:
```suggestion
of this mechanism are out of scope for this document.
```

**Review by chris-wood — APPROVED** [review]
LGTM with some nits

File: draft-irtf-cfrg-opaque.md (hunk 1 above)
Comment (bytemare) [reply draft-irtf-cfrg-opaque.md:323]: I'm not native in English so it's very probable that might suggestion comes from another language.

But does putting the singular to password not mean something like "the one password that all the clients have" instead of "the password of each client"?

File: draft-irtf-cfrg-opaque.md [hunk 4]
```diff
@@ -1923,7 +1923,7 @@ protocols such as TLS.
 The specification as written here differs from the original cryptographic design in {{JKX18}}
 and the corresponding CFRG document {{I-D.krawczyk-cfrg-opaque-03}}, both of which were used
 as input to the CFRG PAKE competition. This section describes these differences, including
-their motivation and explanation as to why they preserve the provable security of OPAQUE based
+their motivation and explanation as to why they preserve the provable security of OPAQUE-based
```
Comment (kevinlewi) [inline draft-irtf-cfrg-opaque.md:1926]: this change should be reverted

File: draft-irtf-cfrg-opaque.md [hunk 5]
```diff
@@ -2085,7 +2085,7 @@ adds domain separation, in particular against servers that choose the same
 OPRF key for multiple clients. The `client_identity` as input to the OPRF
 also acts as a key identifier that would be required for a proof of the
 protocol in the multi-key setting; the OPAQUE analysis in {{JKX18}} assumes
-single server-key instances. Adding `server_identity`` to the OPRF input
+single server-key instances. Adding `server_identity` to the OPRF input
```
Comment (kevinlewi) [inline draft-irtf-cfrg-opaque.md:2088]: this change should be reverted

File: draft-irtf-cfrg-opaque.md [hunk 6]
```diff
@@ -2233,7 +2233,7 @@ message for an unregistered client if these client enumeration attacks can
 be mitigated through other application-specific means or are otherwise not
 applicable for their threat model.
 
-OPAQUE does not prevent against this type of attack during the registration flow.
+OPAQUE does not prevent this type of attack during the registration flow.
```
Comment (kevinlewi) [inline draft-irtf-cfrg-opaque.md:2236]: I think a better change would be "protect against"

File: draft-irtf-cfrg-opaque.md (hunk 1 above)
Comment (kevinlewi) [reply draft-irtf-cfrg-opaque.md:323]: I think it is best left signular, agree with Chris's suggestion here

File: draft-irtf-cfrg-opaque.md [hunk 7]
```diff
@@ -329,7 +329,7 @@ as a "compiler" for transforming any suitable AKE protocol into a secure
 aPAKE protocol. (See {{security-considerations}} for requirements of the
 OPRF and AKE protocols.) This document specifies one OPAQUE instantiation
 based on {{TripleDH}}. Other instantiations are possible, as discussed in
-{{alternate-akes}}, but their details are out of scope for this document.
+{{alternate-akes}}, but their details are out of the scope of this document.
```
Comment (kevinlewi) [inline draft-irtf-cfrg-opaque.md:332]: I would revert this change

**Review by kevinlewi — CHANGES_REQUESTED** [review]

**Review by kevinlewi — APPROVED** [review]
I went ahead and addressed the comments I left with a commit, so I am approving now!
